src/daemons/StorageDaemon.cpp

```diff
@@ -166,8 +166,10 @@ int main(int argc, char *argv[]) {
     schemaMan->init(metaClient.get());
 
     LOG(INFO) << "Starting Storage HTTP Service";
-    nebula::WebService::registerHandler("/status", [] {
-        return new nebula::storage::StorageHttpHandler();
+    nebula::WebService::registerHandler("/storage", [&kvstore] {
```

Comment:
Just for your information, whenever you pass or capture smart pointers by pointers or references, be caureful on the ownerships.